namespace CustomException
{
    internal class Program
    {
        private static void Main(string[] args)
        {
            //TestingOfException2.TestingJustThrowException();
            //TestingOfException2.TestingThrowExException();
        }
    }
}